index.js

```diff
@@ -60,3 +60,25 @@ const showError = (e) => {
   email.value = '';
 };
 form.addEventListener('submit', showError);
+
+// This is for local storage
+const userContent = {
+  fullName: '',
+  email: '',
+  textArea: '',
+};
+
+form.addEventListener('change', () => {
+  userContent.fullName = document.querySelector('#full-name').value;
+  userContent.email = document.querySelector('#email').value;
+  userContent.textArea = document.querySelector('#textarea').value;
+
+  localStorage.setItem('stored', JSON.stringify(userContent));
+});
+
+const storedObject = JSON.parse(localStorage.getItem('stored'));
+if (storedObject) {
+  document.querySelector('#email').value = storedObject.email;
+  document.querySelector('#full-name').value = storedObject.lastName;
```

Comment:
- [ ] The name is `undefined` after the page reloads, kindly consider checking your variable names, it may have a typo, I think `storedObject.lastName` should be `storedObject.fullName`, this will make the inputs fill with the correct values after reloading the page